app/controllers/concerns/flash_i18n.rb

```diff
@@ -0,0 +1,30 @@
+module FlashI18n
+  protected
+
+  def redirect_to(url, options = {})
+    self.class._flash_types.each do |flash_type|
```

Comment:
And here's where I get a little bit confused about the necessity for the translations. If there's a user need driving it, then ignore everything I'm about to say;

This "feels railsy" enough to confuse new developers into thinking it's a standard rails feature, but it isn't, so it feels dangerous to be doing so -- adding a layer of indirection that isn't immediately evident to the developer (because they'll think "Neat, i didn't know rails could do that") and proceed past, without realising there's code here that could be causing a bug they're seeing.